### ISSUE TITLE
FrontHistory does not provide data

### ISSUE THREAD

**Issue by EscolarProgramming** [ApparyllisOrg/SimplyPluralApi#246]
Hello all,
Unfortunately, the FrontHistory does not seem to provide any data, or I am using the method incorrectly. I am trying to call the FrontHistory from another system. For this I use the following URL: `https://api.apparyllis.com:8443/v1/frontHistory/XXXXXXXXX?startTime=1681814528&endTime=1681900928`

No matter what values I enter at startTime / endTime, I always get back an empty array. The way I read the API, a number is needed, but neither the epoch timestamp nor the timestamp in milliseconds returns data.

**Comment by Cellaryllis** [comment]
Hey, you can't get the front history of another system. If you want to get the fronters of another system you can use:

`v1/friends/getFrontValues`
To get a string of the current fronters

and

`/v1/friend/:id/getFront"`
To get a list of current fronters

**Comment by SparkliTwizzl** [comment]
i'm encountering this issue on both dev and production when i try to get either global or individual front history for my own accounts. i even resorted to passing 2000-01-01 at 00:00:00 and datetime.now() as the start and end times, no effect. i only ever got en empty list returned for the front history.

**Comment by Cellaryllis** [comment]
> i'm encountering this issue on both dev and production when i try to get either global or individual front history for my own accounts. i even resorted to passing 2000-01-01 at 00:00:00 and datetime.now() as the start and end times, no effect. i only ever got en empty list returned for the front history.

Are you passing seconds or milliseconds? If you pass seconds then it won't return anything because the time input is interpreted as milliseconds. 

If you want to see a working example you can open the app in the browser, open dev tools and check the network logs to see what kind of requests the app sends and then mimic those on your project.

**Comment by SparkliTwizzl** [comment]
i'm passing `datetime.datetime( 2021, 1, 1, 0, 0, 0, 0 ).timestamp()` as the startTime parameter argument and `datetime.datetime.now().timestamp()` as the endTime parameter argument (Python) and get an empty list returned. i copied a frontHistory GET request from the network logs in browser and manually sent it with code, which worked. i chalk this up to the API documentation being extremely sparse, it would benefit from including these kinds of details.

tldr i guess i wasnt formatting my requests correctly

**Comment by Cellaryllis** [comment]
I agree the documentation is sparse, but as the single developer maintaining the app, SDK, API, servers, user support and API documentation, API documentation is at the very bottom of priorities. We're happy to accept PRs on the documentation site (Which is public as a repo on the Apparyllis account) if you want to update it with more specifics.

**Comment by SparkliTwizzl** [comment]
ah, good to know, i might just do that then.
